pom.xml

```diff
@@ -71,22 +77,21 @@
       <dependency>
         <groupId>org.jenkins-ci.plugins.workflow</groupId>
         <artifactId>workflow-step-api</artifactId>
-        <version>2.16</version>
+        <version>2.21</version>
       </dependency>
       <dependency>
         <groupId>org.hamcrest</groupId>
         <artifactId>hamcrest-core</artifactId>
-        <version>1.3</version>
+        <version>2.2</version>
```

Comment:
Can probably be removed, or excluded from dependencies pulling it. In 2.x, hamcrest-core is included in hamcrest so no longer necessary.